controllers/pipeline/pipeline_suite_test.go

```diff
@@ -65,8 +65,7 @@ var _ = BeforeSuite(func() {
 			filepath.Join("..", "..", "config", "crd", "bases"),
 			filepath.Join(
 				build.Default.GOPATH,
-				"pkg", "mod", "github.com", "tektoncd",
-				"pipeline@v0.40.2", "config",
+				"pkg", "mod", appstudiotest.GetRelativeDependencyPath("tektoncd/pipeline"), "config",
```

Comment:
This and all other changes to the `GetRelativeDependencyPath` calls will be superseded by #75 because the `test` package has been moved from the release-service repository into the standalone [operator-goodies](https://github.com/redhat-appstudio/operator-goodies/) repository.

Reply:
I think it's probably best to remove these changes from this PR and rebase once #75 is merged

Reply:
Thanks! Updated.